content/integrations/docker.md

```diff
@@ -19,7 +19,7 @@ Get metrics from Docker in real time to:
 
 # Installation
 
-There are three ways to setup the Docker integration: install the agent on the host, on a single privileged container, and on each individual container.
+There are two ways to setup the Docker integration: install the agent on the host or on a single sidecar container.
```

Comment:
s/agent/Agent

Also, what is 'the host'? Many readers will know, but it's better to be explicit: 'Install the Agent on any container-running host, or run the docker-dd-agent container.'